README.md

```diff
@@ -10,10 +12,38 @@ For resilience we adopt the "Let it crash" model which the telecom industry has
 
 Actors also provide the abstraction for transparent distribution and the basis for truly scalable and fault-tolerant applications.
 
-Akka is Open Source and available under the Apache 2 License.
+Learn more at [akka.io](http://akka.io/).
```

Comment:
should we also add direct links to the (html) docs for `current` Java and Scala docs?